roles/edpm_libvirt/molecule/default/verify.yml

```diff
@@ -13,38 +13,46 @@
         # common directories
         - "/etc/tmpfiles.d/"
         - "/var/lib/openstack"
-        - "/var/lib/openstack/config/containers"
-        - "/var/log/containers"
-        - "/var/log/containers/stdouts"
         # extrenal deps
         - "/var/lib/openstack/config/ceph"
         # libvirt directories
         - "/var/lib/libvirt"
-        - "/var/lib/openstack/config/libvirt"
-        - "/var/log/containers/libvirt"
+        - "/etc/libvirt"
     - name: ensure firewall is configured
       ansible.builtin.include_tasks: "test-helpers/verify_firewall.yaml"
     - name: ensure ceph secret is configured
       ansible.builtin.include_tasks: "test-helpers/verify_ceph_secret.yaml"
     - name: ensure systemd services are defined and functional
       ansible.builtin.include_tasks: "{{test_helper_dir}}/verify_systemd_unit.yaml"
       loop:
-        - { "name": "edpm_libvirt_virtlogd.service" }
-        - { "name": "edpm_libvirt_virtnodedevd.service" }
-        - { "name": "edpm_libvirt_virtproxyd.service" }
-        - { "name": "edpm_libvirt_virtqemud.service" }
-        - { "name": "edpm_libvirt_virtsecretd.service" }
-    - name: ensure podman container exists and are running
-      ansible.builtin.include_tasks: "{{test_helper_dir}}/verify_podman.yaml"
-      loop:
-        - "libvirt_virtlogd"
-        - "libvirt_virtnodedevd"
-        - "libvirt_virtproxyd"
-        - "libvirt_virtqemud"
-        - "libvirt_virtsecretd"
+        # rhel installed services have service units defiend via rpm in
```

Comment:
could we use the service units shipped by the RHEL package and remove our customizations?

Reply:
we are i have removed our custom services and we are using the ones provide by the rhel rpm which are not in /etc/systemd which is why i left this comment

Reply:
to be clear the only time we are not using the libvirt packaged service files for libvirt is libvirt_guest and that is because we only use the shutdown part of libvirt-guest not the start up part so we have to override it